resources/game_data/resource_generator/resource_generator.gd

```diff
@@ -16,6 +16,9 @@ class_name ResourceGenerator
 @export var column: int = 0
 @export var dynamic_efficiency_id: String
 
+@export var sfx_generate: AudioStream
+@export var sfx_yield: AudioStream
+
```

Comment:
Not happy with the names for these attributes, could use some ideas

generate is for when you attempt to harvest resource (i.e. button down)
yield is for when you successfully attempt to harvest resource (button up + success)

button up + failure defaults to progress buttons' default up

Reply:
I think naming them related to their signals is the way.

On a related note, I'd suggest to avoid export's, see my comments and let me know what you think about that approach? 👀